### ISSUE TITLE
Fine tuning model by my own custom dataset.

### ISSUE THREAD

**Issue by sangtv9** [TreB1eN/InsightFace_Pytorch#175]
Thanks for you great repo. I used pretrained model for my dataset. But, It does not reach good enough accuracy. So I must re-train the model by my dataset. But I only reach 65% at accuracy on validation dataset. The model loss only decrease to approximate 10. Have anyone met the issue? Have you had any idea to deal this? Thanks very much.

**Comment by sangtv9** [comment]
This is my config for training 
SGD (                                                                                                                   Parameter Group 0                                                                                                           dampening: 0                                                                                                            lr: 0.05                                                                                                                momentum: 0.9                                                                                                           nesterov: False                                                                                                         weight_decay: 0.0004                                                                                                                                                                                                                        Parameter Group 1                                                                                                           dampening: 0                                                                                                            lr: 0.05                                                                                                                momentum: 0.9                                                                                                           nesterov: False                                                                                                         weight_decay: 0                                                                                                     )                
I use CrossEntropy for loss function

**Comment by momohuangsha** [comment]
@sangtv9    Hey! Has your problem been solved?  whta is your backbone network? I have the same problem (backbone:mobileface)

**Comment by ANDRESHZ** [comment]
Hi @momohuangsha and @sangtv9, I hope you are having nice results, I am wondering if you finally found a way to tune with your dataset.

Can you share with the community your code?
